src/ngraph/node.hpp

```diff
@@ -329,9 +319,6 @@ namespace ngraph
                                                                bool for_get_output_element = true);
 
         /// Checks that there is exactly one output and returns its shape
-        // TODO: deprecate in favor of node->output(0).get_shape() with a suitable check in the
-        // calling code, or updates to the calling code if it is making an invalid assumption of
-        // only one output.
         const Shape& get_shape() const;
```

Comment:
This really should be deprecated since it is not clear what shape you are referring to. I prefer get_input_shape(index)/get_output_shape(index) methods as nice, clear helper functions.